src/Strategy/Elixir/MixTree.hs

```diff
@@ -233,62 +250,64 @@ mixTreeParser = concat <$> ((try (mixDep 0) <|> ignoredLine) `sepBy` eol) <* eof
     mixDep depth = do
       _ <- takeP (Just "spacing ~ one of (`), (|), ( )") (1 + depth * 4)
 
+      -- Dependency's name
       dep <- chunk "--" *> space *> findName <* space
-      version <- optional . try $ do findRequirement
 
-      location <- between (char '(') (char ')') parseDepSCM
+      -- Dependency's version
+      version <- optional . try $ findRequirement
+
+      -- Dependency's scm
+      scm <- between (char '(') (char ')') parseDepSCM
       _ <- takeWhileP (Just "ignored") (not . isEndLine)
 
-      deps <- many $ try $ mixRecurse $ depth + 1
-      pure [MixDep (PackageName dep) (toDependencyVersion =<< version) location (concat deps)]
+      -- Deep Dependencies
+      deps <- many $ try $ mixRecurse (depth + 1)
+      pure [MixDep (PackageName dep) (toDependencyVersion =<< version) scm (concat deps)]
```

Comment:
[optional nit]: prefer `>>=` to `=<<`.  It's a little unintuitive considering that most similar operators go the other direction, but it does seem to be more idiomatic in other haskell libraries.
```suggestion
      pure [MixDep (PackageName dep) (version >>= toDependencyVersion) scm (concat deps)]
```

Reply:
+in style guide: https://github.com/fossas/team-analysis/issues/671